packages/jss/src/utils/escape.js

```diff
@@ -2,14 +2,12 @@ const env = process.env.NODE_ENV
 
 const escapeRegex = /([[\].#*$><+~=|^:(),"'`])/g
 
+const nativeEscape = typeof CSS === 'undefined' ? CSS.escape : undefined
```

Comment:
Should be `typeof CSS !== 'undefined'`

Reply:
Also could be `const escape = typeof CSS !== 'undefined' ? CSS.escape : (str) => str.replace(escapeRegex, '\\$1')`

That way we would just call `escape(str)` then.

Reply:
already fixed the first one, we need an additional wrapper around this escape function to avoid escaping in production

Reply:
actually I need to shift as much as possible inside of that wrapping function, because it will be removed from the bundle

Reply:
yeah, but instead of doing the check if nativeEscape exists we would just call a function then

Reply:
Why will it be removed from the bundle?

Reply:
because of the if production check

Reply:
```js
const escape = typeof CSS !== 'undefined' ? CSS.escape : str => str.replace(escapeRegex, '\\$1')

export default str => {
  // We don't need to escape it in production, because we are not using user's
  // input for selectors, we are generating a valid selector.
  if (env === 'production') return str
  
  return escape(str)
}
```